### ISSUE TITLE
Add Launching a Jupyter Lab session on Nightingale

### ISSUE THREAD

**Issue by lhelms2** [ncsa/Nightingale_real#77]
See instructions from Weddie in Nightingale Slack channel from 10/3. Work with @pmenstrom as tester.

**Comment by lhelms2** [comment]
@pmenstrom here's an initial build: https://docs.ncsa.illinois.edu/systems/nightingale/en/jupyterlab/user_guide/software.html#launching-a-jupyterlab-session
Please test out the steps and let me know how it goes.

**Comment by pmenstrom** [comment]
In #1 they will need to use the bastion as a jump host
ssh -J <your_username>@ngale-bastion-1.ncsa.illinois.edu <your_username>@ng-<login_node>

Need a blank line after #3

#4 Got this prompt
Are you sure you want to continue connecting (yes/no/[fingerprint])?
Answered "y" and then was prompted for my password again. The first password was for the bastion host, second is for the login node. I think they will only get the "yes/no" question the first time.

#5 Delete the "and" from "in the first terminal and into the web browser"

**Comment by lhelms2** [comment]
Submitted for review/merge in PR #81

**Comment by lhelms2** [comment]
PR is merged into main.
